spec/features/course/assessment/assessment_attempt_spec.rb

```diff
@@ -93,6 +101,32 @@
         end
       end
 
+      scenario 'I can view tabbed assessments and tabs if assessment has more than 1 question' do
+        assessment_tabbed_single_question
+        visit course_assessments_path(course)
+
+        within find(content_tag_selector(assessment_tabbed_single_question)) do
+          find_link(
+            I18n.t('course.assessment.assessments.assessment_management_buttons.attempt'),
+            href: course_assessment_submissions_path(course, assessment_tabbed_single_question)
+          ).click
+        end
+
+        expect(page).not_to have_selector('ul.nav.nav-tabs.tab-header')
+
+        assessment_tabbed
+        visit course_assessments_path(course)
+
+        within find(content_tag_selector(assessment_tabbed)) do
+          find_link(
+            I18n.t('course.assessment.assessments.assessment_management_buttons.attempt'),
+            href: course_assessment_submissions_path(course, assessment_tabbed)
+          ).click
+        end
+
+        expect(page).to have_selector('ul.nav.nav-tabs.tab-header')
```

Comment:
@weiqingtoh Possible to test when the user switches the tab, the question code window displays correctly ?  I feel it's better to have that spec since there are issues with hidden tab and ACE.